app/resources/api/v2/tag_set_resource.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # Provides a JSON:API representation of {TagSet}.
+    #
+    # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+    # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package for Sequencescape's implementation
+    # of the JSON:API standard.
+    #
+    # This resource represents a TagSet, which links together two related tag groups.
+    # It includes the following attributes and relationships:
+    #
```

Comment:
Would it be possible to include the example GET requests, including the filter examples?

I’ve added example requests as part of updated docs [here](https://github.com/sanger/sequencescape/pull/4687). You can then use the GitHub action to rebuild the Yard Syntax and view these comments/ examples [here](https://sanger.github.io/sequencescape/Api/V2.html)

Reply:
I think you might need to run this [Workflow](https://github.com/sanger/sequencescape/actions/workflows/generate_pages.yml), as described [here](https://github.com/sanger/sequencescape?tab=readme-ov-file#documentation), to update the SS Yard documentation on GitHub pages :)